### PR TITLE
Update farmlands portal guide

### PR DESCRIPTION
**Pull request by IsaacMurtagh**
Changes as requested from farmlands team

### CODE REVIEW

File: src/content/guides/farmlands-portal.mdx [hunk 1]
```diff
@@ -3,11 +3,11 @@ title: Farmlands Portal Guide
 description: Farmlands Co-operative (Farmands) have partnered with Centrapay Payment Gateway (Centrapay) to deliver a secure web portal for Farmlands Card payments.
 nav:
   path: Connections/Farmlands
-  title: Portal Guide
+  title: Portal Introduction Guide
```
Comment (laurencemcknight) [inline src/content/guides/farmlands-portal.mdx:6]: Farmlands asked me to remove 'Introduction' from the title last week. Have they asked for it to be added back in?

File: src/content/guides/farmlands-portal.mdx [hunk 2]
```diff
@@ -19,7 +19,7 @@ Farmlands Co-operative (Farmlands) have partnered with Centrapay Payment Gateway
 
 ### Business Information
 
-The initial setup of your Business and all Branches (referred to as Merchants) will be managed by Farmlands and the email address of the primary Account Owner will be provided to Centrapay.
+The initial setup of your Business and all Branches (referred to as Merchants) will be managed by Farmlands . Farmlands will provide Centrapay with the details of a primary Account Owner for the business who will be loaded by Centrapay. Once setup this account owner will be able login and maintain the other members (users) who have need access to their Account (see [Members](#members)).
```
Comment (laurencemcknight) [inline src/content/guides/farmlands-portal.mdx:22]:
```suggestion
The initial setup of your Business and all Branches (referred to as Merchants) will be managed by Farmlands. Farmlands will provide Centrapay with the details of a primary Account Owner for the business who will be loaded by Centrapay. Once setup this account owner will be able login and maintain the other members (users) who have access to their Account (see [Members](#members)).
```

File: src/content/guides/farmlands-portal.mdx [hunk 3]
```diff
@@ -59,69 +59,67 @@ The Centrapay Business Portal provides a real-time authorisation of a Farmlands
...
 
     > Make sure the correct merchant is selected
 
     ![Enter payment amount on the Centrapay business portal](/farmlands-payment-enter-amount.png)
 
-3. Scan the barcode on the rear of the Farmlands Card by using the camera function, scanner (if available). Alternatively, manually enter the 9-digit Card number located on the front of the Card (bottom left-hand corner or below the barcode on the rear of the Card).
+3. **Scan the barcode** on the rear of the Farmlands Card by using the camera function or scanner (if available). Alternatively, manually enter the 9-digit Card number located on the front of the Card (bottom left-hand corner or below the barcode on the rear of the Card).
 
-    > Enter the 9-digit card number, not the 16-digit card number or 9-digit account number.
+    > The 9-digit Card number displayed on the Card must be entered, not the 16-digit card number or 9-digit account number.
```
Comment (laurencemcknight) [inline src/content/guides/farmlands-portal.mdx:74]: Using "The 9-digit number displayed on the Card must be entered" would be more concise but if this is what Farmlands have suggested then just keep it as is.

File: src/content/guides/farmlands-portal.mdx [hunk 4]
```diff
@@ -145,7 +143,7 @@ From this screen, you can also select an authorisation and action a “release,
 
 ## Setting a Default Merchant
 
-**Select your Default Merchant (First-time user) -** A default merchant can be changed at any time by a user that follows the steps below
+**Select your Default Merchant (First-time user) -** The default merchant is the branch or outlet that the user normally works from, and which the transaction will be associated to.
```
Comment (laurencemcknight) [inline src/content/guides/farmlands-portal.mdx:146]:
```suggestion
**Select your Default Merchant (First-time user) -** The default merchant is the branch or outlet that the user normally works from, and which the transaction will be associated with.
```

File: src/content/guides/farmlands-portal.mdx (hunk 1 above)
Comment (IsaacMurtagh) [reply src/content/guides/farmlands-portal.mdx]: Got it, thanks

File: src/content/guides/farmlands-portal.mdx [hunk 5]
```diff
@@ -213,13 +213,19 @@ If the revised sale amount is:
 - less than the original authorisation amount, submit the amended invoice with the original authorisation short code.
 - greater than the original authorisation sales amount, release the hold on the customer’s funds via the Portal. Once completed, generate a new authorisation request for the revised sales amount.
 
+**How do I release funds held on a customer’s Card?**
+
+Go into the Portal, find the original authorisation by searching for the Short Code. Click the authorisation and click Release Funds.
```
Comment (laurencemcknight) [inline src/content/guides/farmlands-portal.mdx:218]:
```suggestion
Go into the Portal, find the original authorisation by searching for the Short Code. Click the authorisation and click "Release Funds".
```

File: src/content/guides/farmlands-portal.mdx [hunk 6]
```diff
@@ -19,7 +19,7 @@ Farmlands Co-operative (Farmlands) have partnered with Centrapay Payment Gateway
 
 ### Business Information
 
-The initial setup of your Business and all Branches (referred to as Merchants) will be managed by Farmlands and the email address of the primary Account Owner will be provided to Centrapay.
+The initial setup of your Business and all Branches (referred to as Merchants) will be managed by Farmlands . Farmlands will provide Centrapay with the details of a primary Account Owner for the business who will be loaded by Centrapay. Once setup this account owner will be able login and maintain the other members (users) who have or need access to the Account (see [Members](#members)).
```
Comment (laurencemcknight) [inline src/content/guides/farmlands-portal.mdx:22]:
```suggestion
The initial setup of your Business and all Branches (referred to as Merchants) will be managed by Farmlands. Farmlands will provide Centrapay with the details of a primary Account Owner for the business who will be loaded by Centrapay. Once setup this account owner will be able login and maintain the other members (users) who have or need access to the Account (see [Members](#members)).
```

**Review by laurencemcknight — APPROVED** [review]

**Pull request merged**